src/ontology/Makefile

```diff
@@ -244,8 +244,7 @@ reasoned-plus-equivalents.owl: reasoned.owl imports/equivalencies.owl
 $(ONT).owl: reasoned.owl
 	$(ROBOT) merge -i $< annotate -V $(ONTURI)/releases/`date +%Y-%m-%d`/$(ONT).owl -o $@
 
-# obo is self-contained
-$(ONT).obo: $(ONT)-base.owl
+$(ONT).obo: $(ONT).owl
```

Comment:
What is `$(ONT).owl` vs. `$(ONT)-base.owl`?

Reply:
$(ONT).owl (mondo.owl) corresponds to the full ontology you see in Protege - with all the imports. "base" has the imports stripped out.